cmd/sequencer/start/start.go

```diff
@@ -107,7 +107,6 @@ func GetStartRollappCmd(rollappConfig config.RollappConfig, lightNodeEndpoint st
 			"--log-file", filepath.Join(rollappConfigDir, "rollapp.log"),
 			"--log_level", "debug",
 			"--max-log-size", "2000",
-			"--json-rpc.address", "0.0.0.0:8545",
```

Comment:
We need this functionality to allow external machines to connect to the JSON RPC. Please ensure it's exposed in a way that doesn't disrupt the operation of Rollappd

Reply:
Agree, imo, should update the `app.toml` upon RollApp's initialization instead of starting like this.
For current users, guide them to manually update app.toml